legal-api/report-templates/certificateOfAmalgamation.html

```diff
@@ -34,15 +34,15 @@
                         <tr class="no-page-break">
                             <td class="col-50">
                                 <div class="{{'upper-text' if entity.legalName!='Not Available'}}">
-                                    and {{ entity.name }} ({{ entity.identifier }})
+                                    and {{ entity.legalName }} ({{ entity.identifier }})
```

Comment:
this will fail now. it only work for foreign business

Reply:
You can include this change. i will fix it in my pr
![image](https://github.com/bcgov/lear/assets/64869009/15dff186-0c35-4367-8292-843290db2291)